cobalt/android/apk/app/src/main/java/dev/cobalt/coat/CobaltActivity.java

```diff
@@ -504,13 +492,9 @@ protected String[] getArgs() {
 
     // If the URL arg isn't specified, get it from AndroidManifest.xml.
     boolean hasUrlArg = hasArg(args, URL_ARG);
-    // If the splash screen url arg isn't specified, get it from AndroidManifest.xml.
-    boolean hasSplashUrlArg = hasArg(args, SPLASH_URL_ARG);
-    // If the splash screen topics arg isn't specified, get it from AndroidManifest.xml.
-    boolean hasSplashTopicsArg = hasArg(args, SPLASH_TOPICS_ARG);
     // If the Evergreen-Lite arg isn't specified, get it from AndroidManifest.xml.
     boolean hasEvergreenLiteArg = hasArg(args, EVERGREEN_LITE);
-    if (!hasUrlArg || !hasSplashUrlArg || !hasSplashTopicsArg || !hasEvergreenLiteArg) {
+    if (!hasUrlArg || !hasEvergreenLiteArg) {
```

Comment:
Can you leave out any Evergreen for now ? I think it's likely going to end up with its own APK in the end